helpers/GenerateResourcesAndImage.ps1

```diff
@@ -115,7 +130,13 @@ Function GenerateResourcesAndImage {
     $ServicePrincipalClientSecret = $env:UserName + [System.GUID]::NewGuid().ToString().ToUpper();
     $InstallPassword = $env:UserName + [System.GUID]::NewGuid().ToString().ToUpper();
 
-    Connect-AzAccount
+    if (([string]::IsNullOrEmpty($AzureClientId)))
```

Comment:
nitpick: too many parentheses
`if ([string]::IsNullOrEmpty($AzureClientId))`

Reply:
Thank you for the feedback. I have now removed the duplicate parentheses in lines 123 (original code example), 133 and 195. Looks much better! :)